tools/vms-generator/utils/utils.go

```diff
@@ -291,6 +287,10 @@ func addHostDisk(spec *v1.VirtualMachineInstanceSpec, path string, hostDiskType
 
 func GetVMIMigratable() *v1.VirtualMachineInstance {
 	vmi := getBaseVMI(VmiMigratable)
+	// having no network leads to adding a default interface that may be of type bridge on
+	// the pod network and that would make the VMI non-migratable. Therefore, adding a network.
```

Comment:
(no action required) this comment would be clearer as: "Default interface type is bridge is not migratable. Use masquerade."

Reply:
but the default interface type is not necessarily a bridge as it is configurable at the cluster level, right? also I could have interpreted the message you propose as "if you add an interface without a type (e.g., with just a name) we set its type to bridge by default and the VMI is not migratable then" - which is not what I actually mean here. so I prefer to be more explicit here

Reply:
@ahadas you are right, I forgot we allow to configure it now.